README.md

```diff
@@ -1,25 +1,29 @@
-# Getting Started with Create React App
+# Coveo Headless Starter Project
+
+This project serves as a starting point for the Coveo Headless Tutorial (**coming soon!**).
+
+## Getting Started with Create React App
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+### Available Scripts
 
 In the project directory, you can run:
 
-### `npm start`
+#### `npm start`
 
 Runs the app in the development mode.\
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
 The page will reload if you make edits.\
 You will also see any lint errors in the console.
 
-### `npm test`
+#### `npm test`
```

Comment:
If you delete the test file, do we want to remove that section altogether?
Or will we add tests at some point later?

Reply:
Yea, tough question. Basically, the way I see it, the command still works, it just doesn't currently do anything because there are no tests. I'm not against getting rid of it though.

Adding tests at some point is an interesting idea. It might be cool to have tests in the starter project, and give them more tests in the tutorial to add to the project as they make changes to the components. But, I'm not sure how much of that would really relate to Headless, and how much would be more of a "unit testing tutorial", which we'd want to avoid. What do you think?

Reply:
That's an interesting question.

This is a starter project that's not meant to be feature complete, so I think it's going to be quite static. When a new version of Headless comes out that introduces one or more breaking changes, we'll have to make sure we do the necessary adjustments here, and that's about it. That being said, it may be useful for us when we do those kinds of adjustments to be able to ensure that we didn't break anything else in the process.

It's also interesting because if we ask them to add new features during the tutorial, they can verify whether what they're doing is breaking anything else :)

I'm not sure I'd have them add new tests as they work in the tutorial; as you said @kaitlynkatz, this would become a bit of a unit testing tutorial and that's not what we want. Still, overall, it's probably good that they can have a look at how we built our tests.

Perhaps we could ask someone from the Search UI team what they think.

Reply:
I like the idea of having UTs for people to validate that their code works properly :D

I agree that we shouldn't have readers create UTs in the tutorial. It's just that if we leave the command in the readme without having any real UT it's distracting IMO. If we add UTs ourselves then it would make sense to leave the command I think. It would actually do something, and people would have a scaffold to start writing more involved tests themselves if they want.